src/main.rs

```diff
@@ -73,6 +71,20 @@ fn cert_entries() -> Result<Vec<CertTableEntry>> {
     }
 }
 
+fn cert_chain_url() -> Result<String> {
```

Comment:
`cert_chain` usually implies VCEK + CA chain to me. Would you prefer to call this `ca_chain_url`?